elmslie-android/src/main/kotlin/money/vivid/elmslie/android/ElmStoreLazy.kt

```diff
@@ -104,25 +99,3 @@ class RetainedElmStore<Event : Any, Effect : Any, State : Any>(
     const val StateBundleKey = "elm_store_state_bundle"
   }
 }
-
-class RetainedElmStoreFactory<Event : Any, Effect : Any, State : Any>(
```

Comment:
since 2.9.0 we don't need it any more [link](https://developer.android.com/jetpack/androidx/releases/lifecycle#2.9.0-alpha09)